tests/test_yaml_file_types.py

```diff
@@ -0,0 +1,930 @@
+from contextlib import contextmanager
+
+import pytest
+from ruamel.yaml.constructor import DuplicateKeyError
+
+from komodo.yaml_file_types import (
+    KomodoException,
+    PackageStatusFile,
+    ReleaseFile,
+    RepositoryFile,
+    _komodo_error,
+)
+
+VALID_PYTHON_PACKAGE_KV_LIST = 'zopfli: "0.3"\npytest: 0.40.1'
```

Comment:
I can't find repeated usage of these constants, I think I would just inline these strings in the tests below. Keeping them as variables could serve a documentation purpose, but it seems that is already covered by usage of the `id` argument to `pytest.param`.